src/ucp/wireup/select.c

```diff
@@ -538,7 +538,8 @@ ucp_wireup_add_lane_desc(const ucp_wireup_select_info_t *select_info,
     lane_desc->proxy_lane   = proxy_lane;
     lane_desc->dst_md_index = dst_md_index;
     lane_desc->lane_types   = UCS_BIT(lane_type);
-    for (lane_type_iter = 0; lane_type_iter < UCP_LANE_TYPE_LAST;
+    UCS_STATIC_ASSERT(0 == UCP_LANE_TYPE_AM);
+    for (lane_type_iter = UCP_LANE_TYPE_AM; lane_type_iter < UCP_LANE_TYPE_LAST;
```

Comment:
maybe for such enums, add FIRST constant?

Reply:
ok, added FIRST enum value